### PR TITLE
feat(gltf): EXT_feature_metadata - numeric types support

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

File: modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts [hunk 1]
```diff
@@ -127,24 +127,73 @@ function getPropertyDataFromBinarySource(
...
+    | 'UINT32'
+    | 'INT64'
+    | 'UINT64'
+    | 'FLOAT32'
+    | 'FLOAT64'
+    | 'BOOLEAN'
+    | 'STRING'
+    | 'ENUM'
+    | 'ARRAY'
+    | string
```
Comment (ibgreen) [inline modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts:182]: By adding `string` here, all the other constants become meaningless. The resulting type is `string`

File: modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts [hunk 2]
```diff
@@ -170,6 +219,41 @@ function getStringOffsets(
...
+ * Parse numeric property values
+ * @param valuesDataBytes - values data array
+ * @param propertyType - type of the property
+ * @param elementCount - number of rows in the featureTable
+ * @returns Number data in a typed array
+ */
+function getNumericAttributes(
+  valuesDataBytes: Uint8Array,
+  propertyType:
+    | 'INT8'
```
Comment (ibgreen) [inline modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts:232]: This is getting very verbose, you should probably define some types.
```typescript
export type I3SPropertyType = 
   | 'INT8'
    | 'UINT8'
    | 'INT16'
    | 'UINT16'
    | 'INT32'
    | 'UINT32'
    | 'INT64'
    | 'UINT64'
    | 'FLOAT32'
    | 'FLOAT64'
    | 'BOOLEAN'
    | 'STRING'
    | 'ENUM'
    | 'ARRAY';

export type I3SPropertyIntegerType = 
   | 'INT8'
    | 'UINT8'
    | 'INT16'
    | 'UINT16'
    | 'INT32'
    | 'UINT32'
    | 'INT64'
    | 'UINT64'
    | 'FLOAT32'
    | 'FLOAT64;
```

**Review by ibgreen — APPROVED** [review]

File: modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts (hunk 1 above)
Comment (belom88) [reply modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts:182]: It is to comply with the spec

File: modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts (hunk 2 above)
Comment (belom88) [reply modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts:233]: Done

**Pull request merged**